### DISCUSSION TITLE
記事の編集で自動インデントされるのを設定でON/OFFできないでしょうか？

### DISCUSSION THREAD

**Discussion by murachi1208** [increments/qiita-discussions#558] [要望・アイデア・不具合]
記事の編集フォームで ”先頭にスペースがある文章で改行”した場合、自動的にインデントされるのですが
こちらの機能を設定でON/OFFできないでしょうか？

下記のような状態
![image](https://github.com/increments/qiita-discussions/assets/7439378/cfb16e41-4cfc-4800-9cc1-5126b09e75f1)

致命的な問題ではありませんが、個人的にはOFFにしたいときが多いので・・・

**Comment by tanaka-asami415** [comment]
@murachi1208 
ご意見ありがとうございます！

現在の所、インデントの ON/OFF 機能をアップデートする予定はないのですが
現状、Shift＋Enterキーで改行していただくとインデント無しで改行できます。

ただ今後のアップデート内容によっては挙動しなくなる可能性も０ではないため、ご了承いただけると幸いです🙏
